### ISSUE TITLE
make file error

### ISSUE THREAD

**Issue by shuq007** [quickreflex/rtl8188eus#35]
**### The drivers for the device on Linux and Windows are not working**

After using git close command we got these details:
`git clone -b v5.2.2.4 https://github.com/quickreflex/rtl8188eus.git`
`Cloning into 'rtl8188eus'...`
`remote: Enumerating objects: 1312, done.`
`remote: Total 1312 (delta 0), reused 0 (delta 0), pack-reused 1312`
`Receiving objects: 100% (1312/1312), 4.64 MiB | 36.00 KiB/s, done.`
`Resolving deltas: 100% (777/777), done.`

when we use `make all` command we stuck up here and got the following error:
**`/home/admin/rtl8188eus/os_dep/linux/os_intfs.c:1240:22: error: initialization from incompatible pointer type [-Werror=incompatible-pointer-types]
  .ndo_select_queue = rtw_select_queue,
                      ^~~~~~~~~~~~~~~~`**

**Comment by StefanosAstaras** [comment]
Yup, same problem here. It looks like a field was removed from the ndo_select_queue signature in kernels >=5.2 and rtw_select_queue is no longer compatible. I naively tried to remove the same field from the latter, but it's not that simple (compiles but crashes).

**Comment by iandol** [comment]
Same error on 5.3.0-26-generic #28-Ubuntu

**Comment by cstayyab** [comment]
Adding 
`EXTRA_CFLAGS += -Wno-incompatible-pointer-types`
to Makefile solves the problem

**Comment by mriso79** [comment]
The change above in makefile works, but now I have this problem when running make all:

error: macro "access_ok" passed 3 arguments, but takes just 2
  if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {


error: ‘access_ok’ undeclared (first use in this function)
  if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {
